src/constants.py

```diff
@@ -16,3 +16,4 @@
 SYNAPSE_NGINX_PORT = 8080
 SYNAPSE_SERVICE_NAME = "synapse"
 TEST_SERVER_NAME = "server-name-configured.synapse.com"
```

Comment:
I'd rather have these constants defined in a module located under the `tests` directory as they are not part of the sources

Reply:
I have moved them